.github/workflows/tests.yaml

```
@@ -0,0 +1,36 @@
+name: tests
+on: [push, workflow_dispatch]
+jobs:
+    build:
+        runs-on: ubuntu-20.04
+        strategy:
+            matrix:
```

This will let each job run to completion, even if a single one fails.

That helps debug individual jobs, so we can see if some pass and just one or two fails.

```suggestion
        strategy:
            fail-fast: false
            matrix:
```

---

Thanks, yeah, will turn that on temporarily.